weather_mv/loader_pipeline/bq_test.py

```diff
@@ -222,9 +232,34 @@ def test_extract_rows_nan(self):
             'time': '2018-01-02T06:00:00+00:00',
             'u10': None,
             'v10': 0.03294110298156738,
+            'geo_point': geojson.dumps(geojson.Point((-108.0, 49.0))),
         }
         self.assertRowsEqual(actual, expected)
 
+    def test_extract_rows__with_valid_lat_long(self):
+        valid_lat_long = [[-90, -360], [-90, -359], [-45, -180], [-45, -45], [0, 0], [45, 45], [45, 180], [90, 359],
+                          [90, 360]]
+        actual_val = [
+            '{"type": "Point", "coordinates": [0, -90]}',
+            '{"type": "Point", "coordinates": [1, -90]}',
+            '{"type": "Point", "coordinates": [-180, -45]}',
+            '{"type": "Point", "coordinates": [-45, -45]}',
+            '{"type": "Point", "coordinates": [0, 0]}',
+            '{"type": "Point", "coordinates": [45, 45]}',
+            '{"type": "Point", "coordinates": [-180, 45]}',
+            '{"type": "Point", "coordinates": [-1, 90]}',
+            '{"type": "Point", "coordinates": [0, 90]}'
+        ]
+        for x in range(len(valid_lat_long)):
+            expected = fetch_geo_point(valid_lat_long[x][0], valid_lat_long[x][1])
+            self.assertEqual(actual_val[x], expected)
```

Comment:
This looks good! Consider making these all subtests.

https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Reply:
Required code-changes completed.